railties/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+*   Upgrade `jsbundling-rails` to support the source maps.
```

Comment:
we don't need changelog entries for things that were not released previously yet.

Reply:
Ah, David already removed.

Reply:
Gotcha! Thanks